src/vcf/record.cpp

```diff
@@ -184,6 +188,23 @@ namespace ebi
         
     }
     
+    void Record::check_alternate_allele_beginning(std::string const & alternate) const
```

Comment:
To use more concrete terminology, this method could be renamed to `check_alternate_allele_symbolic_prefix`.

"Symbolic alternate allele" is the name specifically used for alternate alleles of the form <SOME_ALT>.